electrode-ota-server-dao-mariadb/src/queries/MetricQueries.ts

```diff
@@ -27,4 +27,8 @@ export const MetricQueries = {
     getMetricsByStatusAfterSpecificTime : `SELECT count(*) as total, app_version, label, previous_deployment_key,
                     previous_label_or_app_version, status FROM metric WHERE deployment_id = ? AND create_time >= ?
                     GROUP BY status, label, app_version, previous_deployment_key, previous_label_or_app_version`,
+
+    getMetricsByStatusAfterSpecificTimeAndVersion : `SELECT count(*) as total, app_version, label, previous_deployment_key,
+                    previous_label_or_app_version, status FROM metric WHERE deployment_id = ? AND create_time >= ? AND app_version in (?,?,?)
+                    GROUP BY status, label, app_version, previous_deployment_key, previous_label_or_app_version`,
```

Comment:
is this query going to be slow?

Reply:
Initial run doesn't show any slowness. But on HTTP, yet to test on PROD.

Reply:
Running the query in PROD DB, following is the stats seen:

- With specific app-version the runtime is around `1.10s`

- Without involving app-version the runtime is around `1.40s`

Reply:
timing is good.  using `explain` is better.